contrib/labstack/echo/echotrace.go

```diff
@@ -0,0 +1,48 @@
+// Package echo provides functions to trace the labstack/echo package (https://github.com/labstack/echo).
+package echo
+
+import (
+	"strconv"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/labstack/echo"
+)
+
+// Middleware returns middleware that will trace incoming requests.
+func Middleware(service string) echo.MiddlewareFunc {
+	return func(next echo.HandlerFunc) echo.HandlerFunc {
+		return func(c echo.Context) error {
+			request := c.Request()
+			resource := c.Path()
```

Comment:
```suggestion
			resource := request.Method + " " + c.Path()
```
per https://github.com/DataDog/dd-trace-go/blob/61d8c99c511e0edf8b27364bd6cc053623720662/contrib/gorilla/mux/mux.go#L101